SignalMessaging/categories/UIView+OWS.h

```diff
@@ -83,6 +83,12 @@ CGFloat ScaleFromIPhone5(CGFloat iPhone5Value);
 - (NSLayoutConstraint *)autoPinLeadingToSuperviewWithMargin:(CGFloat)margin;
 - (NSLayoutConstraint *)autoPinTrailingToSuperview;
 - (NSLayoutConstraint *)autoPinTrailingToSuperviewWithMargin:(CGFloat)margin;
+
+- (NSLayoutConstraint *)autoPinTopToSuperview;
+- (NSLayoutConstraint *)autoPinTopToSuperviewWithMargin:(CGFloat)margin;
+- (NSLayoutConstraint *)autoPinBottomToSuperview;
+- (NSLayoutConstraint *)autoPinBottomToSuperviewWithMargin:(CGFloat)margin;
```

Comment:
Picked this over from hotfix and elaborated a bit.

Reply:
I think these are now in master, I think this branch needs a rebase.

Reply:
Yup, done.